keyboards/lets_split/keymaps/tylerwince/keymap.c

```diff
@@ -0,0 +1,233 @@
+ * |   `  |   1  |   2  |   3  |   4  |   5  |   |   6  |   7  |   8  |   9  |   0  | Del  |
+ * |------+------+------+------+------+-------   -------+------+------+------+------+------|
+ * | Del  |      |      |      |      |      |   | LEFT | DOWN |  UP  |RIGHT |      |  \   |
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+   +------+------+------+------+------+------|
+ * |Things|1PASS | alt  | CMD  |Lower |Space |   |Space |Raise | Next | Vol- | Vol+ | Play |
+ * `------------------------------------------   ------------------------------------------'
+ */
+[_RAISE] = LAYOUT_ortho_4x12( \
+  KC_GRV,       KC_1,          KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,     KC_0,    KC_DEL, \
+  KC_DEL,       _______,       _______, _______, _______, _______, KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT, _______, KC_BSLS, \
+  _______,      _______,       _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, \
+  RALT(KC_SPC), RGUI(KC_BSLS), KC_LALT, KC_LCMD, _LOWER,   KC_SPC,  KC_SPC,  _RAISE,   KC_MNXT, KC_VOLD, KC_VOLU,  KC_MPLY  \
+),
+
+/* Adjust (Lower + Raise)
+ * ,------------------------------------------   ------------------------------------------.
+ * |      | Reset|      |      |      |      |   |      |Qwerty|Colemk|Dvorak|WTLEFT|WTRGHT|
+ * |------+------+------+------+------+-------   -------+------+------+------+------+------|
+ * |      |      |Aud on|Audoff|AGnorm|AGswap|   |WLEFT |WDOWN | WUP  |WRGHT |WBLEFT|WBRGHT|
+ * |------+------+------+------+------+------|   |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |RGBsUP|RGBsDN|RGBvUP|RGBvDN|WFULL |
+ * |------+------+------+------+------+------+   +------+------+------+------+------+------|
+ * |      |      |      |      |      |      |   |      |      |RGBwav|RGBfla|RGBtog|      |
+ * `------------------------------------------   ------------------------------------------'
+ */
+[_ADJUST] =  LAYOUT_ortho_4x12( \
+  _______, RESET,   _______, _______, _______, _______, _______,   _QWERTY,    _COLEMAK,   _DVORAK,    LCA(KC_7), LCA(KC_8), \
+  _______, _______, AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, LCA(KC_H), LCA(KC_J), LCA(KC_K), LCA(KC_L), LCA(KC_U), LCA(KC_I), \
+  _______, _______, _______, _______, _______, _______, _______,   RGB_SAI,   RGB_SAD,   RGB_VAI,   RGB_VAD,   LCA(KC_ENT), \
+  _______, _______, _______, _______, _______, _______, _______,   _______,   RGB_M_K,   RGB_M_B,   RGB_TOG,   _______ \
+)
+
+
+};
+
+void matrix_init_user(void) {
+  #ifdef RGBLIGHT_ENABLE
+  rgblight_enable();
+  rgblight_sethsv(255, 0, 160);
+  #endif
+}
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer